extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -96,4 +97,20 @@ static public function canShowDiscussions( $cityId ) {
 			return false;
 		}
 	}
+
+	private static function addMainPageMetadata( OutputPage $outputPage ) {
+		if ( F::app()->wg->Title->isMainPage() ) {
+			$promoDetails = WikiaDataAccess::cache(
+					wfMemcKey( "site-attribute-promoDetails" ),
```

Comment:
I called this `liftigniterMetadata`. I thought we should connect it to LI in some way but I'm not really passionate about it.